addons/l10n_eg/demo/demo_partner.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+    <data>        
+        <record id="l10n_eg_res_partner_1" model="res.partner">
+            <field name="name">Egyptian Furniture Company</field>
+            <field eval="[Command.set([ref('base.res_partner_category_14'), ref('base.res_partner_category_12')])]" name="category_id"/>
+            <field name="is_company">1</field>
+            <field name="street">15 Khaled Ibn El-Walid St., Gesr El-Suez; Infront of El-Salam Hotel</field>
+            <field name="city">Heliopolis</field>
+            <field name="state_id" ref='base.state_eg_c'/>
+            <field name="zip">95380</field>
+            <field name="country_id" ref="base.eg"/>
+            <field name="email">egypt.furniture2126@example.com</field>
+            <field name="phone">022986151</field>
```

Comment:
Could we put a VAT number?  (not sure how that works in Egypt, same for EG company)